source/dotnet/NuGet.config

```diff
@@ -4,6 +4,7 @@
     <!-- remove any machine-wide sources with <clear/> -->
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="local" value="packages" />
```

Comment:
probably don't want to check this change in

Reply:
we need to drop console app cause we are using locally produced nuget, but we run nuget restore task before the build; it won't work. it's catch 22.

Reply:
removed the locals